
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
a.out is a Catch v2.13.10 host application.
Run with -? for options

-------------------------------------------------------------------------------
Teste vazio
-------------------------------------------------------------------------------
arvore.cpp:81
...............................................................................

arvore.cpp:81: FAILED:
  {Unknown expression after the reported line}
due to a fatal error condition:
  SIGABRT - Abort (abnormal termination) signal

===============================================================================
test cases: 1 | 1 failed
assertions: 2 | 1 passed | 1 failed


ERRORS:
free(): invalid pointer
